### DISCUSSION TITLE
About half-off sales

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#661] [Q&A] [answered]
I've always remarked that half-off sales don't apply to 'naut recruitment or to purchasing new launch pads.  But should it?

I would be inclined to think the developers must have left them out on purpose.  But half-off sales also didn't used to apply to Autopurchase in the VAB either, and that almost certainly wasn't intended so we fixed in back in version .4.8.  Anyone have any thoughts?

**Comment by hweimer** [comment, marked as answer]
I think half-off sales are already powerful enough without applying to pads and recruitment.

**Reply by peyre** [reply]
I thought that might be the consensus.  Worth bringing up though, I thought.
